backend/src/controller/modelController.ts

```diff
@@ -37,32 +36,23 @@ function handleConfigureModel(req: OpenAiConfigureModelRequest, res: Response) {
 	const configId = req.body.configId as MODEL_CONFIG | undefined;
 	const value = req.body.value;
 
-	let updated = null;
-
-	if (configId && value && value >= 0) {
+	if (configId && value !== undefined) {
```

Comment:
Good catch, just checked and this does indeed fix a bug in dev where we couldn't set 0 values for these.